figanos/matplotlib/utils.py

```diff
@@ -540,23 +542,35 @@ def plot_logo(
     loc : string, int or tuple
         Location of text, replicating https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.legend.html.
         If a tuple, must be in axes coordinates.
-    path_png: str or None
-        Path to picture of logo, must be a png.
-        If none, Ouranos logo is used by default.
-    offsetim_kw: dict
-        Arugments to pass to matplotlib.offsetbox.OffsetImage().
+    logo : str, Path, dict, optional
+        A name (str) or Path to picture of logo, or a name of an already-installed logo.
+        If a Path is provided, the logo will be installed and accessible via the 'Path().name' of file.
+        The default logo is the Figanos logo. To install the Ouranos (or another) logo consult the Usage page.
+        Logos must be in 'png' format.
+    \*\*offset_image_kwargs
+        Arguments to pass to matplotlib.offsetbox.OffsetImage().
 
     Returns
     -------
     matplotlib.axes.Axes
     """
+    if offset_image_kwargs is None:
+        offset_image_kwargs = {"alpha": 0.8, "zoom": 0.25}
+
+    logos = Logos()
+    if logo:
+        if isinstance(logo, pathlib.Path):
+            path_png = logo
+        path_png = logos[logo]
+    else:
+        path_png = logos.default
     if path_png is None:
-        path_png = (
-            pathlib.Path(__file__).resolve().parents[1] / "data" / "ouranos_logo_25.png"
```

Comment:
Je pense que @sarahclaude avait fait le  logo_25 avec une résolution plus basse qui fait un effet plus beau. Faire un zoom sur le logo pleine taille, ça fait un logo qui semble de mauvaise qualité.
avant avec 25:
<img width="119" alt="Capture d’écran, le 2023-10-10 à 17 20 30" src="https://github.com/Ouranosinc/figanos/assets/29236239/25eb8182-7901-43d8-9707-5f0d67a0f028">
maintenant:
<img width="106" alt="Capture d’écran, le 2023-10-10 à 17 20 09" src="https://github.com/Ouranosinc/figanos/assets/29236239/407ca191-523f-464f-8610-cb0f465488c9">

Reply:
I wanted to bring this up: so Figanos doesn't use actual scaling; Using zoom will deteriorate the image since it doesn't perform any interpolation (from what I can see in the docs for [OffsetImage](https://matplotlib.org/stable/api/offsetbox_api.html#matplotlib.offsetbox.OffsetImage)), but given that it's a PNG, we should be able to scale it down and/or constrain it to a region with matplotlib (an example of that here: https://stackoverflow.com/questions/16742111/scale-image-in-matplotlib-without-changing-the-axis).

Lowering the logo resolution is a double-edged sword since maybe it looks fine on monitors or in a notebook, but it'll look bad in print (article, poster) form. This is the exact problem that Seaborn deals with using their `set_context()` method.

Matplotlib offers lots of ways of interpolating/rescaling images: https://matplotlib.org/stable/api/image_api.html (more if we consider using `scipy`). Maybe it's worth looking into?

Reply:
sure!

Reply:
I think it could also be changed in this PR since it will remove the low resolution picture

Reply:
You're right (and I've already done it, haha). I found a way forward using scikit-image's resizing algorithms.